var/spack/repos/builtin/packages/cget/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Cget(PythonPackage):
```

Comment:
Rename to py-cget

Reply:
As per the other PRs it's a stand alone tool - one calls `cget` on the command line and never `import * from cget`

https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#choosing-a-package-name